src/document/document.ts

```diff
@@ -256,7 +280,7 @@ export class Document<T> {
       this.changeID = change.getID();
 
       if (this.eventStreamObserver) {
-        this.eventStreamObserver.next({
+        this.eventStreamObserver.nextSync({
```

Comment:
It would be nice to replace `next` with something like `trigger`.

Reply:
If there are many changes to modify it to `trigger`, it would be a good to handle it in the upcoming issue. Modifying entire event system operate synchronously can also be addressed in that issue.

Reply:
I have a question.

Why should the entire event system be synchronous?

Reply:
Currently, there are some events that operate synchronously and others that operate asynchronously. This requires double-checking whether an event is synchronous or asynchronous when using it. Standardizing all events to operate synchronously would help to reduce this confusion.